livelog/extractor/writer.go

```diff
@@ -0,0 +1,40 @@
+package extractor
+
+import (
+	"encoding/base64"
+	"io"
+	"regexp"
+)
+
+var re = regexp.MustCompile("#((.*?)#)")
+
+type Writer struct {
+	base io.Writer
+	file []byte
+}
+
+func New(w io.Writer) *Writer {
+	return &Writer{w, nil}
+}
+
+func (e *Writer) Write(p []byte) (n int, err error) {
+	card := re.FindStringSubmatch(string(p))
```

Comment:
I was wondering if a small micro optimization to check if the bytes start with the card prefix might make sense.  If we have a high volume of logs it could save us some processing power to check the prefix before evaluation the regular expression.

```
if bytes.HasPrefix(p, `#((`) == false {
  return e.base.Write(p)
}
```

Reply:
The above check didnt' work for me?  What's the purpose of the brackets?
`	if bytes.HasPrefix(p, []byte("#")) == false {
		return e.base.Write(p)
	}`

Reply:
the aboves works